src/components/assets/EvmNativeToken.vue

```diff
@@ -1,69 +1,115 @@
+            {{ isTruncate ? $n(truncate(bal, 3)) : Number(bal) }}
+          </span>
+          <span class="column--symbol text--symbol">{{ nativeTokenSymbol }}</span>
+        </div>
+      </div>
+
+      <div class="row__actions">
+        <router-link v-if="isZkEvm" :to="buildEthereumBridgePageLink()">
+          <button class="btn btn--icon">
+            <astar-icon-bridge />
+          </button>
+          <span class="text--mobile-menu">{{ $t('assets.bridge') }}</span>
+          <q-tooltip>
+            <span class="text--tooltip">{{ $t('assets.bridge') }}</span>
+          </q-tooltip>
+        </router-link>
+
+        <router-link :to="buildTransferPageLink(nativeTokenSymbol)">
+          <button class="btn btn--icon">
+            <astar-icon-transfer />
+          </button>
+          <span class="text--mobile-menu">{{ $t('assets.send') }}</span>
+          <q-tooltip>
+            <span class="text--tooltip">{{ $t('assets.send') }}</span>
+          </q-tooltip>
+        </router-link>
+
+        <!-- Only SDN is able to bridge via cBridge at this moment -->
+        <a
+          v-if="nativeTokenSymbol === 'SDN'"
+          :href="cbridgeAppLink"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <button class="btn btn--icon">
+            <astar-icon-bridge />
+          </button>
+          <span class="text--mobile-menu">{{ $t('assets.bridge') }}</span>
+          <q-tooltip>
+            <span class="text--tooltip">{{ $t('assets.bridge') }}</span>
+          </q-tooltip>
+        </a>
+
+        <a v-if="isZkatana" :href="faucetSethLink" target="_blank" rel="noopener noreferrer">
+          <button class="btn btn--icon">
+            <!-- TODO: will move icon to AstarUI -->
+            <svg
+              width="24"
+              height="24"
+              viewBox="0 0 24 24"
+              fill="none"
+              xmlns="http://www.w3.org/2000/svg"
             >
-              <button class="btn btn--sm">
-                {{ $t('assets.bridge') }}
-              </button>
-            </a>
-            <a v-if="isZkatana" :href="faucetSethLink" target="_blank" rel="noopener noreferrer">
-              <button class="btn btn--sm">
-                {{ $t('assets.faucet') }}
-              </button>
-            </a>
-            <button
-              v-else-if="isFaucet"
-              class="btn btn--sm"
-              @click="handleModalFaucet({ isOpen: true })"
+              <path
+                d="M11.9 17.61C11.18 17.66 10.36 17.44 9.47999 16.95C8.62999 16.47 8.07999 15.61 7.86999 14.4C7.81999 14.16 7.71999 13.97 7.55999 13.83C7.19999 13.51 6.63999 13.54 6.31999 13.91C6.13999 14.12 6.07999 14.37 6.13999 14.68C6.45999 16.39 7.23999 17.63 8.45999 18.37C9.55999 19.04 10.64 19.37 11.69 19.37C11.79 19.37 11.89 19.37 11.99 19.37C12.24 19.35 12.44 19.25 12.6 19.09C12.77 18.92 12.85 18.72 12.85 18.48C12.85 18.21 12.76 17.99 12.57 17.82C12.39 17.66 12.16 17.59 11.89 17.61H11.9Z"
```

Comment:
Does it make sense to create svg file instead hadrdcoding path in the component? If we move svg to a separate file we have a reusable component

Reply:
I've moved svg icon to Astar UI and replaced it in this [commit](https://github.com/AstarNetwork/astar-apps/pull/956/commits/ee35c19b4b8422c31510093c4f2735c7b75bf87e)